nuxt-frontend/components/jobs/thank-you/LatestBlog.vue

```diff
@@ -1,50 +1,46 @@
 <template>
-  <section class="container -mt-20 md:-mt-12 md:mt-0 md:mb-36">
-    <div class="flex flex-col items-center mb-7 m-auto w-full">
-      <h2
-        class="text-[1.875rem] md:text-[3.4375rem] leading-[2.8125rem] md:leading-[5.15625rem] text-center text-black-core/[0.87] font-inter-bold"
-      >
+  <section class="container mt-16 lg:mb-60">
+    <div class="flex flex-col items-center m-auto w-full">
+      <h2 class="text-center mobile-header-2 lg:desk-header-2 text-black-87">
         Latest Blogs
       </h2>
     </div>
 
     <!-- MOBILE UI START  -->
 
-    <div class="block lg:hidden flex flex-col md:flex-row md:mt-16">
+    <div class="mt-6 block lg:hidden flex flex-col">
       <div
-        class="relative flex flex-col items-center md:items-start md:justify-center md:flex-[60%] md:ml-[5%]"
+        class="relative flex flex-col items-center md:items-start md:justify-center"
       >
         <div
           v-for="(blog, index) in blogs"
           :key="blog"
           @mouseover="activeIndex = index"
           @mouseleave="activeIndex = index"
           @touchstart.passive="activeIndex = index"
-          class="block mb-5 w-full border border-black-core/[0.6] rounded-[20px] box-border p-4"
+          class="block mb-4 w-full border border-black-8 rounded-2xl box-border p-4"
         >
           <div
             class="flex flex-col w-full"
             @click="openBlog(blog.link, 'tap_blog_post')"
           >
-            <div class="block mt-4 w-full">
+            <div class="block w-full">
               <img
                 :src="blog.thumbnail"
-                class="object-cover rounded-lg cursor-pointer"
+                class="object-cover rounded-xl cursor-pointer"
                 loading="lazy"
                 :alt="blog.title"
               />
             </div>
             <div
-              class="flex flex-row justify-between my-2.5 font-inter-regular text-[0.875rem] leading-[1.3125rem] text-black-core/[0.6]"
+              class="flex flex-row justify-between mt-3 sub-h3-regular text-black-60"
             >
               <span class="w-auto">
                 {{ blog.author }}
```

Comment:
if it's a single line then write directly into the `div`, no need to write into `span`